tests/integration/api/test_activation.py

```diff
@@ -151,26 +154,103 @@ async def test_delete_activation_instance(
 ):
     foreign_keys = await _create_activation_dependent_objects(client, db)
 
-    query = sa.insert(models.activation_instances).values(
-        name="test-activation",
-        rulebook_id=foreign_keys["rulebook_id"],
-        inventory_id=foreign_keys["inventory_id"],
-        extra_var_id=foreign_keys["extra_var_id"],
+    query = (
+        sa.insert(models.activation_instances)
+        .values(
+            name="test-activation",
+            rulebook_id=foreign_keys["rulebook_id"],
+            inventory_id=foreign_keys["inventory_id"],
+            extra_var_id=foreign_keys["extra_var_id"],
+        )
+        .returning(models.activation_instances.c.id)
```

Comment:
The last inserted primary key can be retrieved via `inserted_primary_key` attribute of a `Result` object. No need to call `.returning` explicitly to return primary key only.

https://docs.sqlalchemy.org/en/14/core/connections.html#sqlalchemy.engine.CursorResult.inserted_primary_key